### PR TITLE
MAUI-5440-Dev-UG content for Calendar identifier support

### PR DESCRIPTION
**Pull request by Priyadharshini-Periyasamy**
### **Description**

- Added the UG content for Calendar identifier support.

### CODE REVIEW

File: MAUI/Calendar/calendar-identifier.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+layout: post
+title: Calendar identifier in .NET MAUI Calendar control | Syncfusion
+description: Learn here all about the Calendar identifier in Syncfusion .NET MAUI Calendar (SfCalendar) control and more details.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Calendar identifier in .NET MAUI Calendar (SfCalendar)
+The .NET MAUI Calendar supports Calendar Identifier to display the different types of calendars based on various cultures and regions. The supported identifies are Gregorian, Hijri, Korean, Persian, Taiwan, ThaiBuddhist, and UmAlQura. Change the calendar identifier by using the [Identifier] property in the SfCalendar.
```
Comment (florenceliliana) [inline MAUI/Calendar/calendar-identifier.md:11]: The .NET MAUI Calendar supports Calendar Identifier to display the different types of calendars based on various cultures and regions. The supported identifies are Gregorian, Hijri, Korean, Persian, Taiwan, ThaiBuddhist, and UmAlQura. Change the calendar identifier using the [Identifier] property in the SfCalendar.

File: MAUI/Calendar/calendar-identifier.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+* HebrewCalendar<br>
+* JulianCalendar<br>
+* JapaneseCalendar<br>
+* Yoruba calendar<br>
+</td>
+</tr>
+</table>
+
+N>
+* The Flow Direction will be updated based on the calendar identifier. All the DateTime values such as DisplayDate, MinimumDate, MaximumDate, SelectedDate, SelectedDates, SelectedDateRange, SelectableDayPredicate, and SpecialDayPredicate can be given in two ways.
```
Comment (florenceliliana) [inline MAUI/Calendar/calendar-identifier.md:47]: * The Flow Direction will be updated based on the calendar identifier. All the DateTime values, such as DisplayDate, MinimumDate, MaximumDate, SelectedDate, SelectedDates, SelectedDateRange, SelectableDayPredicate, and SpecialDayPredicate, can be given in two ways.

**Review by florenceliliana — CHANGES_REQUESTED** [review]
The review is completed.

File: MAUI/Calendar/calendar-identifier.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: Calendar identifier in .NET MAUI Calendar control | Syncfusion
```
[the remark this answers is not in the document]
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/calendar-identifier.md]: Content review updated

File: MAUI/Calendar/calendar-identifier.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+description: Learn here all about the Calendar identifier in Syncfusion .NET MAUI Calendar (SfCalendar) control and more details.
```
[the remark this answers is not in the document]
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/calendar-identifier.md]: Content review updated

File: MAUI/Calendar/calendar-identifier.md (hunk 1 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/calendar-identifier.md]: Content review updated

File: MAUI/Calendar/calendar-identifier.md (hunk 2 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/calendar-identifier.md]: Content review updated

File: MAUI/Calendar/calendar-identifier.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+* JulianCalendar<br>
+* JapaneseCalendar<br>
+* Yoruba calendar<br>
+</td>
+</tr>
+</table>
+
+N>
+* The Flow Direction will be updated based on the calendar identifier. All the DateTime values, such as DisplayDate, MinimumDate, MaximumDate, SelectedDate, SelectedDates, SelectedDateRange, SelectableDayPredicate, and SpecialDayPredicate, can be given in two ways.
+* The DateTime instance without specifying calendar identifier. The calendar will handle the DateTime value for the specified calendar identifier.
```
Comment (Sasikumar3595) [inline MAUI/Calendar/calendar-identifier.md:48]: Don't use calendar type - use calendar identfier

Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/calendar-identifier.md:48]: Changed the calendar type to calendar identifier.

File: MAUI/Calendar/calendar-identifier.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+SfCalendar calendar = new SfCalendar();
+calendar.Identifier = CalendarIdentifier.Hijri;
+this.Content = calendar;
+
+{% endhighlight %}
+{% endtabs %}
+
+### DateTime values in Calendar Types
+The DateTime values of DisplayDate, MinimumDate, MaximumDate, SelectedDate, SelectedDates, SelectedDateRange, SelectableDayPredicate, and SpecialDayPredicate can be given in two ways when the calendar identifier is specified other than the GregorianCalendar.
```
[the remark this answers is not in the document]
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/calendar-identifier.md]: Improved the Content

File: MAUI/Calendar/calendar-identifier.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+layout: post
+title: Calendar Identifier in .NET MAUI Calendar Control | Syncfusion
+description: Learn here all about the Calendar identifier in the Syncfusion .NET MAUI Calendar (SfCalendar) control and more details.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Calendar identifier in .NET MAUI Calendar (SfCalendar)
+The .NET MAUI Calendar supports Calendar Identifier to display the different types of calendars based on various cultures and regions. The supported identifies are Gregorian, Hijri, Korean, Persian, Taiwan, ThaiBuddhist, and UmAlQura. Change the calendar identifier using the [Identifier] property in the SfCalendar.
```
Comment (Sasikumar3595) [inline MAUI/Calendar/calendar-identifier.md:11]: The supported identifi**ers** are

Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/calendar-identifier.md]: Comment addressed

**Pull request closed without merging**